### ISSUE TITLE
Loading button animation no longer working

### ISSUE THREAD

**Issue by mr-kojo** [yunity/karrot-frontend#988]
I noted that the quasar buttons don't show this `loader` animation anymore (which is triggered by the `statusMixin`). 

@tiltec, might this be the result of the quasar update?

**Comment by tiltec** [comment]
Indeed, the API changed: http://quasar-framework.org/components/button.html#Button-with-Progress
I can update this, but it might be a nice starter task too :)
